src/components/Form.tsx

```diff
@@ -287,7 +291,6 @@ const Form: React.FC<{
                     <div className="text-right">
                       {toTokenInfo?.decimals && (
                         <NumericFormat
-                          disabled={true}
```

Comment:
Some of the props from the input `NumericFormat` should also be added to this output field now that it can have a value. For example: `placeholder={'0.00'}`